example-site/components/FormExampleMultiStep/FormExampleMultiStepActions.tsx

```diff
@@ -10,15 +10,8 @@ import { Fragment } from 'react';
 export const FormExampleMultiStepActions = () => {
 	const [isModalOpen, openModal, closeModal] = useTernaryState(false);
 
-	const {
-		hasCompletedPreviousSteps,
-		isSubmittingStep,
-		saveAndExit,
-		isSavingBeforeExiting,
-		cancel,
-	} = useFormExampleMultiStep();
-
-	if (!hasCompletedPreviousSteps) return null;
```

Comment:
We don't need to do this check anymore since it's handled in `FormExampleMultiStepContainer`